src/cluster_asm.c

```diff
@@ -216,6 +246,30 @@ int asmDebugIsFailPointActive(int channel, int state) {
     return 0;
 }
 
+sds asmGenInfoString(sds info) {
+    int active_tasks = 0;
+
+    listIter li;
+    listNode *ln;
+    listRewind(asmManager->tasks, &li);
+    while ((ln = listNext(&li)) != NULL) {
+        asmTask *task = listNodeValue(ln);
+        if (task->operation == ASM_IMPORT ||
+            (task->operation == ASM_MIGRATE && task->state != ASM_FAILED))
+        {
+            active_tasks++;
+        }
+    }
+
+    return sdscatprintf(info,
```

Comment:
```suggestion
    return sdscatprintf(info ? info : sdsempty(),
```
Maybe the function can handle the word `cat`.

somethings like `catAsmInfoString(info)`?

Reply:
as we start with `asm` prefix in general, I made it `asmCatInfoString()` :)

Reply:
besides, do you consider the case where info is NULL? my suggestion above

Reply:
okay I missed that part. Added now.